docs/concepts/storage/persistent-volumes.md

```diff
@@ -219,6 +219,13 @@ Current reclaim policies are:
 * Delete -- associated storage asset such as AWS EBS, GCE PD, Azure Disk, or OpenStack Cinder volume is deleted
 
 Currently, only NFS and HostPath support recycling. AWS EBS, GCE PD, Azure Disk, and Cinder volumes support deletion.
+In case of Retain policy, admin can make a volume as reusable by following manual reclamation steps.
```

Comment:
"When the Retain reclaim policy is specified, you can manually reclaim your volume with the following steps"